spec/cc/analyzer/issue_spec.rb

```diff
@@ -16,72 +16,44 @@ module CC::Analyzer
         },
         "remediation_points" => 10,
         "type" => "issue",
-      }.to_json
-      issue = Issue.new(output)
+      }
+    end
+
+    it "allows access to keys as methods" do
+      issue = Issue.new(output.to_json)
 
       expect(issue.respond_to?("check_name")).to eq true
       expect(issue.check_name).to eq("Rubocop/Style/Documentation")
     end
 
     describe "#fingerprint" do
+      include EnvHelpers
```

Comment:
Looks like I missed this line in the spec_helper which loads all of the helpers:

```ruby
Dir.glob("spec/support/**/*.rb").each(&method(:load))
```